docs/_includes/ds6/select.html

```diff
@@ -0,0 +1,70 @@
+<div id="select">
```

Comment:
Note that clicking the icon doesn't expand the select (testing in Safari).

Reply:
I will look into this ... the other browsers don't attach any click capabilities to a pseudo-element (as near as I can tell).

Reply:
I know there was a fix in ds4 for this same thing. Can't remember off the top of my head what it was....

Reply:
I wasn't able to research and fix. If you know the fix, or where to find it quick, please let me know.

Reply:
Aaaaaaand, I just had to think about it and remembered `pointer-events: none;`. Magic! 🌤 🎉